android/src/test/java/com/browserstack/AppiumTest.java

```diff
@@ -0,0 +1,27 @@
+package com.browserstack;
+
+import java.net.URL;
+
+import org.openqa.selenium.MutableCapabilities;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.android.options.UiAutomator2Options;
+
+
+public class AppiumTest {
+
+    public AndroidDriver driver;
+
+    @BeforeMethod(alwaysRun=true)
+    public void setUp() throws Exception {
+        MutableCapabilities capabilities = new UiAutomator2Options();
+        driver = new AndroidDriver(new URL("http://127.0.0.1:4723/wd/hub"),capabilities);
```

Comment:
do we want to add source if missing?

Reply:
Keeping consistent with parent selenide repo for now.